### DISCUSSION TITLE
Dolby Vision to HDR10+ transcoding support

### DISCUSSION THREAD

**Discussion by wyup** [cdgriffith/FastFlix#537] [Ideas]
Hello,

I'd like to ask support for Doby Vision to HDR10+ transcoding, given that ffmpeg libpacebo library supports Dolby Vision decoding, according to dovi_tool and hdr10plus_tool quietvoid, he [states](https://github.com/quietvoid/hdr10plus_tool/discussions/67) it is actually possible to parse a file source for histogram information necessary to build HDR10+ compliant metadata.

There are many owners of Samsung tvs that would like such a tool for HDR10+ conversion, and FastFlix being the ideal tool for it.

**Comment by cdgriffith** [comment]
That is a great idea, and there are several things around Dolby Vision I wish we could do. 

However, using tools outside of Dobly's free professional tools such as Metafier and Mezzinator may constitute a breach of their TOS, or at minimum may require a [trim license (costs $1000)](https://visionlicense-us.dolby.com/products/dolby-vision-trims-license). 

This is not something I would add to FastFlix without explicit permission from them directly.

**Reply by wyup** [reply]
Thanks Chris,
What about a tool to generate HDR10+ metadata by analyzing scenes and their individual brightness levels?
